packages/react/package.json

```diff
@@ -94,17 +94,17 @@
     "react-scripts": "3.4.1",
     "react-test-renderer": "16.13.1",
     "rimraf": "3.0.2",
-    "rollup": "2.28.2",
+    "rollup": "2.67.2",
     "rollup-plugin-includepaths": "0.2.4",
     "rollup-plugin-postcss": "3.1.8",
     "rollup-plugin-terser": "7.0.2",
     "ts-jest": "^26.0.1",
     "typed-css-modules": "0.6.4",
-    "typescript": "3.9.3",
     "yup": "^0.32.9"
   },
   "dependencies": {
     "@babel/runtime": "7.11.2",
+    "@emotion/styled-base": "^11.0.0",
```

Comment:
@emoition/styled-base was dependencys dependency, but I had to add it as a direct dependency. Reason was that the code could not find it (or its types) as it was before.